src/main/java/seedu/duke/log/LogLevels.java

```diff
@@ -0,0 +1,5 @@
+package seedu.duke.log;
+
+public enum LogLevels {
```

Comment:
Maybe you can consider adding Javadoc comments here to add clarity to what log levels are